### DISCUSSION TITLE
Home Assistant OS 10.5

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2695] [Announcements]
## Open Virtual Appliance

* Linux: Update kernel 6.1.45 (#2685) @agners

## Generic x86-64

* Linux: Update kernel 6.1.45 (#2685) @agners
* Including the RTW8821ce driver module to support Wifi on the AK1 PRO micro PC (#2659) @dfreimuth

## Hardkernel ODROID

* Linux: Update kernel 6.1.45 (#2685) @agners
* Use the correct Supervisor machine during build (#2681) @agners
* Use Rockchip binaries from official repository (#2673) @agners

## ASUS Tinker

* Linux: Update kernel 6.1.45 (#2685) @agners

## Khadas VIM Series

* Linux: Update kernel 6.1.45 (#2685) @agners

## Generic aarch64

* Linux: Update kernel 6.1.45 (#2685) @agners

**Comment by andyshinn** [comment]
Anyone upgrade a Yellow + NVMe drive without issues?

**Reply by agners** [reply]
I run my production on Yellow with NVMe, for the foreseeable future there won't be a release where that hasn't been tested by me :sweat_smile:

**Comment by Maaang** [comment]
Trying to update to 10.5 from 10.4 running on Generic x86-64. Upon confirming to install the update, Hass OS restarts, but after restart I am still on version 10.4. I would love to add logs, but fail to find anything telling anything about what's going on. Can anyone guide me?

**Reply by sedlotom** [reply]
Same problem

**Reply by agners** [reply]
We are using a A/B type of system: There are two installations of the operating system, and when an upgrade fails, it will fall back to the previous one.

Now there could be various issues why you end up with the old version:
1. The installation of the new OS might fail in the first place
2. The new installed OS might fail to boot completely and therefor cause a fallback
3. The fallback mechanism is not working correctly

The reason any of those three can be quite various as well. From hardware issues to unfortunate disk setup (e.g. if an old disk with HAOS has been attached to the system, such problems appear).

Unfortunately it is not quite straight forward to debug these type of issues. Best is if you setup SSH access to the operating shell (port 22222). The guide [SSH access to the host](https://developers.home-assistant.io/docs/operating-system/debugging#ssh-access-to-the-host) explains the step required to setup such an SSH access.

Once you have access to the OS shell, please try to run the update again. After the restart, run the following commands and post the output:

```
lsblk --output NAME,MAJ:MIN,RM,SIZE,RO,TYPE,MOUNTPOINTS,LABEL,PARTLABEL
journalctl --list-boots
journalctl -u rauc.service
```

**Reply by KortgatGH** [reply]
Same problem as Maaang. I tried the commands above via SSH but after the first one I get a response:
"zsh: command not found: lsblk"
